app/main.cpp

```diff
@@ -1,7 +1,22 @@
 #include "app.h"
 
+using namespace vsite::oop::v4;
+
 // read animal id's (until 0) and display species name
 // display total leg count
 int main()
 {
+	unsigned uID;
+	leg_counter countLegs = leg_counter();
```

Comment:
Ne razumijem, zašto ne samo 
`leg_counter countLegs;`
Ovako napravite dva objekta.